R/XcmsExperiment.R

```diff
@@ -47,6 +47,18 @@
 #'   `keepChromPeaks` (by default `TRUE`), `keepAdjustedRtime` (by default
 #'   `FALSE`) and `keepFeatures` (by default `FALSE`).
 #'
+#' - `c`: multiple `XcmsExperiment` objects can be combined into one using the
+#'   `c()` function. This requires however that all the `XcmsExperiments`'
+#'   `Spectra` objects use the same type of `MsBackend` and that their
+#'   processing queues are empty. Also, only combining of peak detection
+#'   results is supported. Any eventually present alignment or correspondence
+#'   results will be dropped before combining the `XcmsExperiment` objects.
+#'   Finally, at present, only the MS data of the individual `XcmsExperiment`
+#'   objects is combined and any data eventually present in the `@qdata`,
```

Comment:
I haven't dug into the function, but what prevents `c()`ing also slots like `@experimentFiles` from two or more `XcmsExperiment`s ?

Reply:
Honestly - I never used the `@experimentFiles` slot - and am actually also not quite sure it will be used a lot...